src/pretix/plugins/sendmail/tasks.py

```diff
@@ -179,6 +186,11 @@ def send_mails_to_orders(event: Event, user: int, subject: dict, message: dict,
             except SendMailException:
                 failures.append(o.email)
 
+    for chunk in _chunks(objects, 1000):
+        orders = Order.objects.filter(pk__in=chunk, event=event)
+        for o in orders:
```

Comment:
as far as I understand we could prevent Django from keeping all orders matched by the QuerySet in memory by using [QuerySet.iterator()](https://docs.djangoproject.com/en/dev/ref/models/querysets/#iterator)?
```suggestion
        for o in orders.iterator():
```

Reply:
there's no prefetching here (yet) though